code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,591 @@
+openapi: 3.0.3
+info:
+  title: Device location subscription event API
+  description: Service Enabling Network Function API which create, retrieve and delete subscriptions for events related to the location of a device
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
+  - three_legged: []
+ 
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: location-subscription/v0
+        description: Base path for the device location subscription API
+paths:
+  /event-subscriptions:
+    post:
+      tags:
+        - Location Event Subscription
+      summary: 'Create a location event subscription for a device'
+      operationId: createEventSubscription
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateEventSubscription'
+        required: true
+      callbacks:
```

Comment:
We did not have  right now a decision for using call back or providing a standard endpoint for Notification.
There is is an issue in commonalities to discuss this https://github.com/camaraproject/Commonalities/issues/8